docs/source/Usage.md

```diff
@@ -145,6 +145,7 @@ The following is the list of the available analyzers you can run out-of-the-box:
 * `InQuest_REPdb`: Search in [InQuest Lab's](https://labs.inquest.net/repdb) Reputation Database
 * `InQuest_DFI`: Deep File Inspection by [InQuest Labs](https://labs.inquest.net/dfi)
 * `XForceExchange`: scan an observable on [IBM X-Force Exchange](https://exchange.xforce.ibmcloud.com/)
+* `FireHol_IPList`: check if an IP is in [FireHol's IPList](https://iplists.firehol.org/)
```

Comment:
not correct, you just integrated their highest priority list.

Reply:
I believe we can write something like: 
> check if an IP is in `firehol_level1` list by [FireHol's IP List](https://iplists.firehol.org/).

Reply:
Gotcha!

Reply:
I did not change the sentence since we are fulfilling it with the new code changes. I added about the `list_names` parameter in Advanced Usage docs. Please suggest if the sentence needs to be changed.